electrode-ota-server-routes-acquisition/src/handlers.js

```diff
@@ -17,6 +19,18 @@ const handles = {
         const snakeCaseRequested = /update_check/.test(action);
         // if snake_case request? convert to camelCase
         const qs = snakeCaseRequested ? keysToCamelOrSnake(request.query) : request.query;
+        // validate query-parameters
+        missingParameter(qs.deploymentKey, `Deployment key missing`);
+        missingParameter(qs.appVersion, `appVersion missing`);
```

Comment:
do u know if there are any existing /updateCheck calls that has missing appVersion/deploymentKey?

Reply:
wee already have a check here: https://github.com/scrajesh/electrode-ota-server/blob/master/electrode-ota-server-model-acquisition/src/acquisition.js#L30

It's just moved a bit up.